posthog/management/commands/update_batch_export_schedules.py

```diff
@@ -0,0 +1,48 @@
+import datetime as dt
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+
+from posthog.batch_exports.service import sync_batch_export
+from posthog.models import BatchExport
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class Command(BaseCommand):
+    help = "Updates the Temporal schedule for a batch export"
+
+    def add_arguments(self, parser):
+        # TODO - add a way to update all batch exports
+        parser.add_argument("--batch-export-id", default=None, type=str, help="Batch export ID to update")
```

Comment:
Stretch goal: Add a way to update a subset of batch exports. Slice by `team_id` and by `destination.type` I think would be pretty common.